landa/water_body_management/doctype/stocking_target/stocking_target.js

```diff
@@ -17,6 +14,19 @@ frappe.ui.form.on("Stocking Target", {
 			})
 		);
 	},
-});
+	onload: function (frm) {
+		const today = new Date();
+		const current_year = today.getFullYear();
+
+		let target_year;
+		if (today.getMonth() < 10) {
+			target_year = current_year;
+		}
+		else {
+			target_year = current_year + 1;
+		}
+		frm.set_value('year', target_year);
+	}
+	});
```

Comment:
```suggestion
	onload: function (frm) {
		if (frm.is_new() && !frm.doc.year) {
			const today = new Date();
			const current_year = today.getFullYear();
			const current_month = today.getMonth();

			frm.set_value("year", current_month < 10 ? current_year : current_year + 1);
		}
	},
});
```

Das können wir hier eigentlich genau so so wie beim **Catch Log Entry** machen, oder?